core/src/cs/eng1/piazzapanic/customer/CustomerManager.java

```diff
@@ -182,17 +237,87 @@ public void addStation(SubmitStation station) {
     }
 
     public void generateCustomer() {
-        // implement random generation of two or three customers at once here
-        customerQueue.addFirst(
-            new Customer(possibleRecipes[random.nextInt(4)], this)
-        );
+        Integer customerObjective = findAvailableObjective();
+        if (customerObjective != null) {
```

Comment:
Considering we need to implement multiple customers spawning at once, do we plan to just have it not be able to generate more than there currently are available spaces in stations at once?